src/main/app/claims/models/hearingDurationType.ts

```diff
@@ -5,5 +5,6 @@ export enum HearingDurationType {
   TWO_HOURS = 'Two hours',
   HALF_HOUR = '30 minutes',
   ONE_AND_HALF_HOUR = 'One and half hour',
-  ONE_HOUR = 'One hour'
+  ONE_HOUR = 'One hour',
+  TWO_AND_HALF_HOURS = 'Two and half hours'
```

Comment:
To keep the time consistent, this should be placed on line 5, rather than line 9.- So that it is between three and two hours.

Reply:
ONE_DAY = 'One day',
  FOUR_HOURS = 'Four hours',
  THREE_HOURS = 'Three hours',
  TWO_AND_HALF_HOURS = 'Two and half hours',
  TWO_HOURS = 'Two hours',
  ONE_AND_HALF_HOUR = 'One and half hour',
  ONE_HOUR = 'One hour',
  HALF_HOUR = '30 minutes',